account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -186,20 +186,19 @@ def _prepare_payment_line(self, payment, line):
         state = 'normal'
         communication = line.ref or '-'
         if line.invoice:
-            if line.invoice.type in ('in_invoice', 'in_refund'):
-                if line.invoice.reference_type == 'structured':
-                    state = 'structured'
-                    communication = line.invoice.reference
-                else:
-                    if line.invoice.reference:
-                        communication = line.invoice.reference
-                    elif line.invoice.supplier_invoice_number:
-                        communication = line.invoice.supplier_invoice_number
-            else:
-                # Make sure that the communication includes the
-                # customer invoice number (in the case of debit order)
-                communication = line.invoice.number.replace('/', '')
+            if line.invoice.reference_type == 'structured':
                 state = 'structured'
+                # Fallback to invoice number to keep previous behaviour
+                communication = line.invoice.reference or line.invoice.number
+            else:
+                if line.invoice.type in ('in_invoice', 'in_refund'):
+                    communication = (
+                        line.invoice.reference or
+                        line.invoice.supplier_invoice_number or line.ref)
+                else:
+                    # Make sure that the communication includes the
+                    # customer invoice number (in the case of debit order)
+                    communication = line.invoice.number
```

Comment:
I think this comes from the old times before XML files. I personally see this very annoying to have `FV/2016/1234`, and find in the statements `FV20161234`. Harder to read and no automatic matching.